checkstyle-style.xml

```diff
@@ -16,8 +16,6 @@
 	    <!-- Make the @SuppressWarnings annotations available to Checkstyle -->
 	    <module name="SuppressWarningsHolder" />
 
-        <module name="FileContentsHolder" />
```

Comment:
This is [removed in new release](http://checkstyle.sourceforge.net/releasenotes.html#Release_8.2). `FileContentsHolder` is not required any more and just need to be removed from clients configurations.